L1Trigger/L1THGCalUtilities/python/hgcalTriggerNtuples_cfi.py

```diff
@@ -72,36 +72,36 @@
 
 ntuple_triggersums = cms.PSet(
     NtupleName = cms.string('HGCalTriggerNtupleHGCTriggerSums'),
-    TriggerSums = cms.InputTag('hgcalConcentratorProducer:HGCalConcentratorProcessorSelection'),
+    TriggerSums = cms.InputTag('l1tHGCalConcentratorProducer:HGCalConcentratorProcessorSelection'),
 )
 
 ntuple_econdata = cms.PSet(
     NtupleName = cms.string('HGCalTriggerNtupleHGCConcentratorData'),
-    ConcentratorData = cms.InputTag('hgcalConcentratorProducer:HGCalConcentratorProcessorSelection'),
+    ConcentratorData = cms.InputTag('l1tHGCalConcentratorProducer:HGCalConcentratorProcessorSelection'),
 )
 
 ntuple_clusters = cms.PSet(
     NtupleName = cms.string('HGCalTriggerNtupleHGCClusters'),
-    Clusters = cms.InputTag('hgcalBackEndLayer1Producer:HGCalBackendLayer1Processor2DClustering'),
-    Multiclusters = cms.InputTag('hgcalBackEndLayer2Producer:HGCalBackendLayer2Processor3DClustering'),
+    Clusters = cms.InputTag('l1tHGCalBackEndLayer1Producer:HGCalBackendLayer1Processor2DClustering'),
+    Multiclusters = cms.InputTag('l1tHGCalBackEndLayer2Producer:HGCalBackendLayer2Processor3DClustering'),
     FilterClustersInMulticlusters = cms.bool(False)
 )
 
 from L1Trigger.L1THGCal.egammaIdentification import egamma_identification_histomax
 ntuple_multiclusters = cms.PSet(
     NtupleName = cms.string('HGCalTriggerNtupleHGCMulticlusters'),
-    Multiclusters = cms.InputTag('hgcalBackEndLayer2Producer:HGCalBackendLayer2Processor3DClustering'),
+    Multiclusters = cms.InputTag('l1tHGCalBackEndLayer2Producer:HGCalBackendLayer2Processor3DClustering'),
     EGIdentification = egamma_identification_histomax.clone(),
     FillLayerInfo = cms.bool(False),
     FillInterpretationInfo = cms.bool(True)
 )
 
 ntuple_towers = cms.PSet(
     NtupleName = cms.string('HGCalTriggerNtupleHGCTowers'),
-    Towers = cms.InputTag('hgcalTowerProducer:HGCalTowerProcessor')
+    Towers = cms.InputTag('l1tHGCalTowerProducer:HGCalTowerProcessor')
 )
 
-hgcalTriggerNtuplizer = cms.EDAnalyzer(
+L1THGCalTriggerNtuplizer = cms.EDAnalyzer(
```

Comment:
```suggestion
l1tHGCalTriggerNtuplizer = cms.EDAnalyzer(
```
This should be lower-case.

Reply:
Fixed